charmcraft.yaml

```diff
@@ -19,6 +19,11 @@ parts:
   charm:
     build-packages:
       - libpq-dev
+      - libffi-dev
```

Comment:
Maybe adding like this would work?
```
charm-binary-python-packages:
    - psycopg2-binary
    - cryptography
```

Reply:
thanks, this is much nicer. I updated charmcraft.yaml to use this instead

Reply:
thanks, this is much nicer. I updated charmcraft.yaml to use this instead